v3-sandbox/prototype/sdk/src/main/java/org/hiero/sdk/simple/internal/network/key/KeyFactory.java

```diff
@@ -101,15 +32,10 @@ public static PrivateKey createPrivateKey(@NonNull final byte[] privateKey) {
      * @throws IllegalArgumentException if the bytes do not match the algorithm
      */
     @NonNull
-    public static PrivateKey createPrivateKey(@NonNull final byte[] privateKey, @NonNull final KeyAlgorithm algorithm) {
+    public static PrivateKey createPrivateKey(@NonNull final KeyAlgorithm algorithm, @NonNull final byte[] privateKey) {
```

Comment:
So we're enforcing that the user needs to know what type of key they're inputting? Do we not want to allow the SDK to determine the key type itself based on DER encoding?